game/theater/start_generator.py

```diff
@@ -322,7 +328,15 @@ def generate_armor_at(self, position: PointWithHeading) -> None:
             self.control_point,
         )
 
-        group = generate_armor_group(self.faction_name, self.game, g)
+        player_wants_shorad = self.generator_settings.shorads_in_armor_groups
+
+        group = generate_armor_group(
+            self.faction_name,
+            self.game,
+            g,
+            shorad,
+            player_wants_shorad,
```

Comment:
This is more simply (after the `PresetLocations` refactor mentioned above) `include_shorad=location.has_shorad and settings_allow_shorad`

Reply:
done